client/src/helpers/fhir/builder/ServiceRequestBuilder.ts

```diff
@@ -33,24 +30,24 @@ const defaultSR = (): Partial<ServiceRequest> => ({
 export class ServiceRequestBuilder {
     private serviceRequest: Partial<ServiceRequest> = defaultSR()
 
-    constructor(serviceRequest: any) {
-      if (serviceRequest != null) {
-        this.serviceRequest = {
+    constructor(serviceRequest?: any) {
+      if (serviceRequest) {
+        this.serviceRequest = JSON.parse(JSON.stringify({
```

Comment:
if only a shallow copy is needed you could do something like : {... }. If you need a deep copy, then I would use lodash utility.